prow/lib-multicluster.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
```

Comment:
@su225 Can we reduce this PR to just the files under the `prow` directory. I'd like to focus the review on just setting up the clusters.

@stevenctl mind taking a look to compare to your work on istio/istio?

Reply:
Ok. I'll move other changes to another PR.